src/test/scala/edu/berkeley/ce/rockslicing/BlockVTKSpec.scala

```diff
@@ -163,12 +165,12 @@ class BlockVTKSpec extends FunSuite {
   test("Connectivities for each face should be in terms of vertex ID's") {
     val vtkBlock = BlockVTK(unitCube)
     val faceConnectivities = Map(
-      face1 -> Seq[Int](7, 6, 5, 4),
-      face2 -> Seq[Int](3, 1, 0, 2),
-      face3 -> Seq[Int](5, 6, 0, 1),
-      face4 -> Seq[Int](7, 4, 3, 2),
-      face5 -> Seq[Int](2, 0, 6, 7),
-      face6 -> Seq[Int](4, 5, 1, 3)
+      face1 -> Seq[Int](7, 1, 3, 6),
```

Comment:
These changes are definitely annoying. I'm not sure about how to make these less brittle. The thing that messes it up is that everything is based on the order in which the vertices appear in the array of vertices. If this changes, the unit tests change too. 
I'm not sure of a practical way to ensure that the vertices are always stored starting from the same one. Do you have any ideas?